product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx

```diff
@@ -4,58 +4,59 @@ navTitle: "Version 5.1.0"
+| BDR       | 5.1.0   | Bug fix         | Fixed join and replication when SDW and standby_slot_names are set. |
+| BDR       | 5.1.0   | Bug fix         | Fixed spurious warnings when processing sequence options. |
+| BDR       | 5.1.0   | Bug fix         | Fixed upgrades for nodes with CRDTs. |
+| BDR       | 5.1.0   | Bug fix         | Adjusted lag tracking parameters for LCRs from pg_stat_replication. |
+| BDR       | 5.1.0   | Bug fix         | Adjusted node routing options defaults based on node kind. This change is related only to the display of the information and not the behavior. For example, witness nodes aren't marked as candidates for receiving writes. |
+| BDR       | 5.1.0   | Bug fix         | All sequences are now converted to distributed during create node. |
+| BDR       | 5.1.0   | Bug fix         | Fixed streaming transactions with `standby_slot_names`. This might have led to a subscriber-only node getting ahead of a data node. |
+| BDR       | 5.1.0   | Bug fix         | Made priority work properly for routing selection. Previously, node priority was working only if there wasn't a previous leader, which is never the case on failover.|
+| BDR       | 5.1.0   | Bug fix         | Fixed the recording of its join as complete for the first node. |
+| BDR       | 5.1.0   | Bug fix         | Disabled tracing by default. Tracing was enabled only for initial debugging and was meant to be disabled before 5.0 release. |
+| BDR       | 5.1.0   | Bug fix         | Added support for reload configuration for the pglogical receiver. When the server receives a reload signal, the pglogical receiver reloads and applies the configuration changes. |
+| BDR       | 5.1.0   | Bug fix         | Improved missing instance error message in `bdr.monitor_group_raft()`. |
+| BDR       | 5.1.0   | Bug fix         | Implemented consistent use of tcp keepalives across all BDR connections. This change added the following GUCs:<br/>`bdr.global_connection_timeout`<br/>`bdr.global_keepalives`<br/>`bdr.global_keepalives_idle`<br/>`bdr.global_keepalives_interval`<br/>`bdr.global_keepalives_count`<br/>`bdr.global_tcp_user_timeout`<br/>The defaults are set to fairly conservative values and are subject to adjustments in the future patches. |
+| BDR       | 5.1.0   | Bug fix         | Closed Raft connections on no activity after a timeout. This uses wal_sender_timeout/wal_receiver_timeout underneath. |
+| BDR       | 5.1.0   | Bug fix         | Made backends that receive Raft messages easily identifiable. Added information in the log message related to Raft backends. |
+| BDR       | 5.1.0   | Bug fix         | Fixed issue whereby Parallel Apply might slow down under heavy load.
+| BDR       | 5.1.0   | Enhancement     | Restarting sync workers is now avoided. This fix is to prevent the node join from failing when config changes are made that signal the restart of subscription workers. |
```

Comment:
```suggestion
| BDR       | 5.1.0   | Enhancement     | Restarting sync workers is now avoided.<br/>This fix is to prevent the node join from failing when config changes are made that signal the restart of subscription workers. |
```